src/components/TopicMenu/TopicMenu.js

```diff
@@ -192,7 +192,13 @@ class TopicMenu extends PureComponent {
             <div
               className={`wkp-layer-toggler ${collapsed ? 'collapsed' : ''}`}
               style={{
-                display: topic.key === activeTopic.key ? 'block' : 'none',
```

Comment:
Another core change to remove the topic arrow if there aren't any layers listed.